resources/site.rb

```diff
@@ -93,11 +96,13 @@
 end
 
 action :add do
-  if !current_resource.site_id
+  if exists
```

Comment:
updated to new `exists` function so that it is easier to update all at once and positive if condition